File: benchmarks/sum.cpp
#include <iostream>
#include <vector>

int main() {
    std::vector<int> array(100);
    for (int i = 0; i < 100; ++i) array[i] = i + 1;
    int sum = 0;
    for (auto item : array) {
        sum += item;
    }
    std::cout << sum << std::endl;
}

File: benchmarks/sum.cs
// sum.cs
using System;

class Program
{
    static void Main(string[] args)
    {
        // Example C# code
        int sum = 0;
        int[] array = new(100);
        for (int i = 0; i < array.Length; i++)
        {
            sum += array[i];
        }
        Console.WriteLine(sum);
    }
}
